heron/spi/src/java/com/twitter/heron/spi/common/Key.java

```diff
@@ -170,7 +170,10 @@
   SHELL_BINARY          ("heron.binaries.shell",           "${HERON_BIN}/heron-shell"),
   PYTHON_INSTANCE_BINARY("heron.binaries.python.instance", "${HERON_BIN}/heron-python-instance"),
   CPP_INSTANCE_BINARY   ("heron.binaries.cpp.instance",    "${HERON_BIN}/heron-cpp-instance"),
-  DOWNLOADER_BINARY     ("heron.binaries.downloader",      "${HERON_BIN}/heron-downloader");
+  DOWNLOADER_BINARY     ("heron.binaries.downloader",      "${HERON_BIN}/heron-downloader"),
+
+  // keys for `heron` command line
+  UPDATE_PROMPT         ("heron.command.update.prompt", "disabled");
```

Comment:
The config name is not very straghtforward. Suggest to add a comment for it. Also include the values:
disabled => no prompt
prompt => enable prompt.

Reply:
still not very clear. How about let's do something like this:

// Prompt user when more containers are required so that user has another chance to double check quota is available. To enable it, change the config from "disabled" to "prompt".

Reply:
updated